introduction.md

```diff
@@ -5,3 +5,7 @@ sadfdsaf
 asfsadf sadfsdjflksjdafkjfla;sjf asdkfjdsalfjo;
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?